### ISSUE TITLE
Loadbalancer external listener for the kafka service

### ISSUE THREAD

**Issue by vjvel** [confluentinc/cp-helm-charts#602]
Hi, 

I seen there is option to create nodeport service, Is there any option to create LB service for the kafka pods. 

I want to allow external access for the kafka broker. How to set the external advertisment address for kafka broker.

**Comment by vjvel** [comment]
Any one can help on this plz ?

**Comment by OneCricketeer** [comment]
You might have better luck with Strimzi if you want to have LoadBalancer or Ingress resources. 

Direct links to docs - https://strimzi.io/docs/operators/latest/overview.html#configuration-points-broker_str

**Comment by gavasv-sapiens** [comment]
any possibility of this being achieved with the current helms ?? … Any help will be much appreciated …

**Comment by OneCricketeer** [comment]
Yes, these charts can be used, but they don't currently offer such a feature, and are no longer supported, so PR to add such a feature might not be approved.

The Confluent operator is where Kubernetes features are being added. https://github.com/confluentinc/confluent-kubernetes-examples/tree/master/networking/external-access-load-balancer-deploy

But this will require a license after 30 days
